scripts/run_backend_tests.py

```diff
@@ -223,7 +230,8 @@ def get_all_test_targets_from_path(
     paths = []
     excluded_dirs = [
         '.git', 'third_party', 'node_modules', 'venv',
-        'core/tests/data', 'core/tests/build_sources']
+        'core/tests/data', 'core/tests/build_sources',
+        '.direnv']
```

Comment:
I think this should match the .gitignore! Should we create a GitHub issue to have a reusable get_files_in_repo (which internally uses .gitignore) function that can be used here and in other relevant places?

Reply:
Will do after this PR (create an issue or do in follow up PR). Thanks.